lib/matplotlib/figure.py

```diff
@@ -682,11 +682,10 @@ def set_size_inches(self, *args, **kwargs):
         matplotlib.Figure.get_size_inches
         """
 
-        forward = kwargs.get('forward', False)
-        if len(args) == 1:
-            w, h = args[0]
-        else:
-            w, h = args
+        # the width and height have been passed in as a tuple to the first
```

Comment:
A comment here wouldn't help much.  None of our code is using the tuple argument form.  Ideally, set_width and set_height would not be needed at all; they could be handled in set_size_inches using keywords, with both w and h defaulted to None.
Does anything actually use set_width and/or set_height at all?  I imagine that something must use them, or the issue of changing them to go through set_size_inches would not have arisen.  Nevertheless, it seems like this is a place where we have the opportunity for some longer-term deprecation and code condensation to reign in our sprawling API.